Autopilot/PathManager/Src/TelemToPathManager.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * Implements methods for communication with the path manager.
+ * Author: Anthony Bertnyk, Messed with for telemetry use by Gordon Fountain
+ */
+
+#include "TelemToPathManager.hpp"
+
+extern "C"
+{
+#include "cmsis_os.h"
+}
+
+void CommWithPMInit()
+{
+    attitudeDataMailQ = osMailCreate(osMailQ(attitudeDataMailQ), NULL);
```

Comment:
The mail queue name should also be similar to the name of the struct for sending telemetry data to the path manager (TelemToPMData or commandsWithPM).